.github/ISSUE_TEMPLATE/whitelist-request.md

```diff
@@ -11,21 +11,19 @@ about: Games tested and found to have no issues.
 ## System Information
 - GPU: <!-- e.g. RX 580 or GTX 970 -->
 - Driver/LLVM version: <!-- e.g. Mesa 18.2/7.0.0 or nvidia 396.54 -->
-- Kernel version: <!-- e.g. 4.17 -->
+- Distro version: <!-- e.g. 4.17 -->
```

Comment:
Comment with example should be probably changed, maybe to "Ubuntu 18.10".

Reply:
Thanks!